tests/feature/services/ShopService/ShopifyTest.php

```diff
@@ -22,67 +19,86 @@ private function shopifyService()
      */
     public function itCanIndexProducts()
     {
-        $products = $this->shopifyService()->products();
-        $this->assertTrue(Arr::has($products, '0.id'));
+        $amount = 2;
+        $products = $this->shopifyService()->products(['limit' => $amount]);
+
+        $this->assertTrue(count($products) === $amount);
```

Comment:
count($products) is enough, if there is only one product this test will fail, so remove the check with amount